patches/base64-js+1.3.1.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/node_modules/base64-js/index.js b/node_modules/base64-js/index.js
+index f087f5b..c070546 100644
+--- a/node_modules/base64-js/index.js
++++ b/node_modules/base64-js/index.js
+@@ -22,9 +22,9 @@ revLookup['_'.charCodeAt(0)] = 63
+ function getLens (b64) {
+   var len = b64.length
+ 
+-  if (len % 4 > 0) {
+-    throw new Error('Invalid string. Length must be a multiple of 4')
+-  }
++  // if (len % 4 > 0) {
```

Comment:
Try other libs making sure it's not breaking anything

Reply:
Moved from `base64-js` to `https://github.com/niklasvh/base64-arraybuffer/` inside our `react-native-simple-crypto` fork.